idea/src/org/jetbrains/kotlin/idea/intentions/RemoveArgumentNameIntention.kt

```diff
@@ -30,7 +32,9 @@ class RemoveArgumentNameIntention : SelfTargetingRangeIntention<KtValueArgument>
 
         val argumentList = element.parent as? KtValueArgumentList ?: return null
         val arguments = argumentList.arguments
-        if (arguments.asSequence().takeWhile { it != element }.any { it.isNamed() }) return null
+        if (!element.languageVersionSettings.supportsFeature(LanguageFeature.MixedNamedArgumentsInTheirOwnPosition)
+            && arguments.asSequence().takeWhile { it != element }.any { it.isNamed() }
```

Comment:
When `MixedNamedArgumentsInTheirOwnPosition` is enabled, you cannot skip this check. Actually you have to check that all of the previous arguments are not named, or are named but in their own positions

You can take a look at `NamedArgumentCompletion::isOnlyNamedArgumentExpected` to see how this check looks exactly

To check that your implementation is correct, try this test: 

```kt
// COMPILER_ARGUMENTS: -XXLanguage:+MixedNamedArgumentsInTheirOwnPosition
// IS_APPLICABLE: false

fun foo(name1: Int, name2: Int, name3: Int) {}

fun usage() {
    foo(name2 = 2, name1 = 1, <caret>name3 = 3)
}
```

The intention should not be applicable, because first two arguments are not in their own positions